packages/ui/tests/e2e/utils/index.ts

```diff
@@ -38,6 +38,10 @@ export function makeClusterTille({path, page}: {path?: string; page?: string}) {
     return [path, page, CLUSTER_TITLE ?? capitalize(CLUSTER)].filter(Boolean).join(' - ');
 }
 
+export function makeQueryTrackerUrl() {
```

Comment:
It is better to use `makeClusterUrl('queries?param1=xxx&param2=yyy')` instead of the function.